tasktypes/commitalert.py

```diff
@@ -60,6 +62,8 @@ def process_task(self, library, task):
         newest_commit = unseen_upstream_commits[-1]
         existing_job = self.dbProvider.get_job(library, newest_commit.revision)
         if existing_job:
+            self.logger.set_context(library.name, existing_job.id)
+
             if my_ff_version in existing_job.ff_versions:
                 # We've already seen this revision, and we've already associated it with this FF version
                 self.logger.log("We found a job with id %s for revision %s that was already processed for this ff version (%s)." % (
```

Comment:
Line 87 in commitalert.py should set a job_id of "new" to represent we're working on a new job we don't have an id for yet.

Reply:
See also #356 which shouldn't be part of this commit (and isn't urgent at all) but is something I noticed while looking at this code.

Reply:
Okay, added that line.